pkg/features/kube_features.go

```diff
@@ -773,6 +773,13 @@ const (
 	// The user namespace has to be created before running kubelet.
 	// All the node components such as CRI need to be running in the same user namespace.
 	KubeletInUserNamespace featuregate.Feature = "KubeletInUserNamespace"
+
+	// owner: @jiahuif
+	// alpha: v1.21
+	// beta:  v1.22
+	//
+	// Enables Leader Migration for kube-controller-manager and cloud-controller-manager
+	ControllerManagerLeaderMigration featuregate.Feature = "ControllerManagerLeaderMigration"
```

Comment:
Probably deserves a comment here that this is linked/copied from the one in controller-manager.

Or the controller-manager one should have a comment depending on which one we consider the source of truth.

Reply:
Updating now.

Reply:
Note: ` pkg/features/kube_features.go ` is always the source of truth for feature gates.